common_analysis_ip_and_uri_finder/ip_and_uri_finder_analysis.py

```diff
@@ -32,14 +31,11 @@ def __init__(self, yara_ip_rules):
     def _find_addresses_of_specific_format(self, string, address_format, validate=True):
         try:
             yara_matches = self.ip_rules.match(data=string)
-        except Exception as e:
-            logging.error("Could not match yara rules: {} {}".format(exc_info()[0].__name__, e))
+        except Exception:
+            logging.error('Could not match yara rules', exc_info=True)
```

Comment:
This does the same thing as
```suggestion
            logging.exception('Could not match yara rules')
```